prisma/schema.prisma

```diff
@@ -0,0 +1,62 @@
+// This is your Prisma schema file,
```

Comment:
wow the schema and migration is simpler than I expected 🤯

Reply:
Right? It's a _tad_ complex for my liking, but with full user-OAuth account bindings, multi-account sign in and session management it's pretty tiny

Reply:
with a custom adapter you could drop some of these fields as well. we just forward all fields for simplicity on our side. account has multiple potentially unused fields, but you could drop them in your own adapter. the default Prisma adapter is 48LOC, so wouldn't be that much work either. https://github.com/nextauthjs/adapters/blob/next/packages/prisma/src/index.ts